packages/irc2as/index.js

```diff
@@ -484,14 +249,15 @@ IrcToActivityStreams.prototype.input = function (string) {
         nick = (msg[3].length <= 2) ? msg[2] : msg[3];
         if (nick === 'undefined') { break; }
         role = MODES[pos2[1]] || 'member';
-        this._sendPresence(nick, role, channel, time);
+        ase.presence(nick, role, channel);
         break;
 
         /** */
-        default:
-        this.events.emit(EVENT_UNPROCESSED, string);
-        break;
+        // default:
+        // this.events.emit(EVENT_UNPROCESSED, string);
+        // break;
```

Comment:
Why is this commented out? Weren't we talking about this in the last Kosmos call, saying it's good to have?